pages/portfolio.html

```diff
@@ -4,9 +4,90 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Portfolio</title>
+  <link href="/styles/style.css" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/3e8fd03e4e.js"
+  crossorigin="anonymous"></script>
 </head>
 <body>
+<header>
+  <nav>
+    <ul class="top-bottom-links">
+      <li>Portfolio</li>
+      <li><a href="/pages/index.html">Home</a></li>
+      <li><a href="/pages/about.html">About</a></li>
+    </ul>
+  </nav>
+</header>
+<h2>Selected Projects</h2>
+<p>
+<ul>
+  <a href="https://github.com/DynamicDev3000/task-list-api">One-to-many relationships API</a>
+  <ul>
```

Comment:
An a tag or ul tag is not a valid child of a ul. Basically, the only thing allowed to be a direct child is an li tag, which could then nest other tags.

Running your html through a validator would catch this.